templates/pages/QA.html

```diff
@@ -217,7 +217,7 @@
                             <a href="https://www.ease-crc.org">
                                 <img id="ease-funding-logo"
                                 class=""
-                                src="{{ url_for('static', filename='img/ease.png') }}"
+                                src="{{ url_for('static', filename='img/Ease-Ease.svg') }}"
```

Comment:
the naming of the file is odd.
the CRC is called EASE with all uppercase, so the leading uppercase character looks wrong.
Please either use all uppercase or all lowercase.
Also why is it "Ease-Ease" and not just "Ease"?

Reply:
Changed the name and committed it. The file is now called 'ease-crc.svg'